contentctl/objects/lookup.py

```diff
@@ -25,6 +25,7 @@
 class Lookup(SecurityContentObject):
     
     collection: Optional[str] = None
+    # TODO (cmcginley): should fields_list be a CSV string? or a list of strings?
     fields_list: Optional[str] = None
```

Comment:
We do we parse `fields_list` as a CSV string and not a list of strings?

Reply:
This is probably somewhat-legacy code/typing.  In the YMLs, fields is a comma-separated string. We should likley convert all the YMLs to use a list[str].
Additionally, Lookups are actually two different kinds of lookups - CSVLookup and KVStore Lookup. These have have different requirements. When we make the above change, we should consider breaking this into two different classes with Lookup as an Abstract Base Class.

I will leave this comment open for discussion

Reply:
How do we want to track small quality of life improvements like this? GitHub issues? Internally on Jira?

Reply:
See https://github.com/splunk/contentctl/issues/220